### DISCUSSION TITLE
Locale to use for Swedish

### DISCUSSION THREAD

**Discussion by joho1968** [mkdocs/mkdocs#3194] [Q&A]
I want to contribute with a Swedish translation for mkdocs. The official locale is "sv_SE", but I'm not sure if you want me to use just "sv", which is the language code, or the full sv_SE locale, which is often displayed as "Swedish (Sweden)". There is also sv_FI, which is often displayed as "Swedish (Finland)". The language code is, as you can see, still "sv".

:thinking:

**Comment by oprypin** [comment]
I think you'd create these if `sv_FI` had a significantly different translation content than `sv_SE`. I don't think it would be?
So just go for `sv`.

And by that rule, maybe `tr_TR` should've been `tr`?
https://github.com/mkdocs/mkdocs/tree/master/mkdocs/themes/mkdocs/locales

To recap what was mentioned

I'm guessing should be just one translation?

* * https://manpages.ubuntu.com/manpages/bionic/man3/DateTime::Locale::sv_SE.3pm.html
  * https://manpages.ubuntu.com/manpages/bionic/man3/DateTime::Locale::sv_FI.3pm.html


* * https://manpages.ubuntu.com/manpages/bionic/man3/DateTime::Locale::tr_TR.3pm.html
  * https://manpages.ubuntu.com/manpages/bionic/man3/DateTime::Locale::tr_CY.3pm.html

I'm guessing should different translation?

* * https://manpages.ubuntu.com/manpages/bionic/man3/DateTime::Locale::pt_BR.3pm.html
  * https://manpages.ubuntu.com/manpages/bionic/man3/DateTime::Locale::pt_PT.3pm.html

* * https://manpages.ubuntu.com/manpages/bionic/man3/DateTime::Locale::zh_Hans_CN.3pm.html
  * https://manpages.ubuntu.com/manpages/bionic/man3/DateTime::Locale::zh_Hant_TW.3pm.html

**Reply by oprypin** [reply]
I think there's a way to make an "official" or at least codified rule, per https://stackoverflow.com/a/46583144

https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry

If the first part of the locale (such as 'sv-' or 'pt-') is present **including the dash** on this page, then the translations should be disambiguated, otherwise not.

**Reply by oprypin** [reply]
* Opened #3195 to handle this
